apps/hobbies-helsinki/Dockerfile

```diff
@@ -0,0 +1,93 @@
+# =======================================
```

Comment:
We should use the Dockerfile from the root wtih certain changes (start to use `helsinkitest/node:14-slim` instead of `alpine` etc). This can be in separate ticker.

Reply:
deleted in #17